feature-libs/asm/components/customer-emulation/customer-emulation.component.spec.ts

```diff
@@ -3,7 +3,7 @@ import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { I18nTestingModule, User } from '@spartacus/core';
 import { UserAccountFacade } from '@spartacus/user/account/root';
-
+import { MockFeatureLevelDirective } from 'projects/storefrontlib/shared/test/mock-feature-level-directive';
```

Comment:
I believe this should be an absolute path, but probably does not hurt the compilation of the tests.

Reply:
The class is not exported as part of a module.  This is the only valid import path.